en/docs/references/error-catalog.md

```diff
@@ -0,0 +1,1828 @@
+# Error Catalog
+This document describes all the REST API error codes that are used in WSO2 Identity Server. 
+
+## Common errors
+
+<div class="errorcode">
+<table>
+  <thead>
+    <th>Error Code</th>
+    <th>HTTP Status Code</th>
+    <th>Error Message</th>
+    <th>Possible Cause</th>
+  </thead>
+  <tbody>
+  <tr>
+    <td>UE-10000</a></td>
+    <td>400</td>
+    <td>Invalid Request</td>
+    <td>Provided request body content is not in the expected format.</td>
+  </tr>
+    <tr>
+    <td>SE-50000</td>
+    <td>500</td>
+    <td>Unexpected processing error</td>
+    <td>The server encountered an error while serving the request.</td>
+  </tr>
+  </tbody>
+</table>
+</div>
+
+---
+
+## Challenge question management
```

Comment:
Shall we make this consistent with the other h2s on the page --> "Challenge question management errors"